scripts/compactor_runner.py

```diff
@@ -189,6 +189,7 @@ def run(self):
         """
         Run compactor.
         """
+        self._print_and_log("Invoked compactor_runner...");
```

Comment:
Typo: Is it Invoked or Invoke?

Reply:
The compactor_runner is already is invoked at this point